framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -54,8 +88,13 @@ export const CreatableSelectDropdown = ({
   onOptionChange,
   width = '100%',
   variant = 'outline',
+  initialValue = '',
 }: CreatableSelectDropdownProps) => {
-  const [ selectedOption, setSelectedOption ] = useState<Option | null>(null)
+  const initialSelectedOption = useMemo(() => standardOptions.find(
```

Comment:
Well done on fixing the useEffect part.
One problem with this is it's going to iterate over `standardOptions` every time `standardOptions` change, which is not really neccessary.

My suggestion is that you remove the dependencies list, so that this logic would be triggered only once (upon mounting).